Core/OfficeDevPnP.Core/Framework/Provisioning/Connectors/OpenXML/PnPPackageHelper.cs

```diff
@@ -13,33 +12,33 @@ namespace OfficeDevPnP.Core.Framework.Provisioning.Connectors.OpenXML
     /// </summary>
     public static class PnPPackageExtensions
     {
-        public static Byte[] PackTemplate(this PnPInfo pnpInfo)
+        public static MemoryStream PackTemplate(this PnPInfo pnpInfo)
```

Comment:
Hi Mikael,
Instead of changing the signature of the already existing public methods, it would be better to add overloads. In fact, even if I totally get the point and I appreciate your efforts, changing the signature of a public method introduces a breaking change. Thus, could you please keep the old method and add a new overload in order to keep the code backward compatible? Thanks.